releaseNotes/MicrosoftAzureAppConfigurationAspNetCore.md

```diff
@@ -1,6 +1,10 @@
 ## Microsoft.Azure.AppConfiguration.AspNetCore
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Azure.AppConfiguration.AspNetCore)
 
+### 4.0.0-preview - July 23, 2020
+* Added multi-targeting support for .NET Core 3.1. [#173](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/173)
+* Updated `Microsoft.Extensions.Configuration.AzureAppConfiguration` reference to `4.0.0-preview`. See the [release notes](./MicrosoftExtensionsConfigurationAzureAppConfiguration.md) for more information on the changes.
+
```

Comment:
We have a breaking change that users must call `IServiceCollection.AddAzureAppConfiguration()` before they can call `UseAzureAppConfiguration`. We need to call this out.